### PR TITLE
fixed mobile and tablet styling

### PR DESCRIPTION
**Pull request by Nick-Ohman**
Fixes # mobile styling with mission control and child dashboard 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [x] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/styles/less/ChildDashboard.less [hunk 1]
```diff
@@ -101,10 +103,12 @@
...
-      @media @mobile {
-        max-width: 100%;
-        max-height: 100%;
-      }
+      max-width: 100%;
+      max-height: 100%;
+      // @media @tablet, @mobile {
+      //   max-width: 100%;
+      //   max-height: 100%;
+      // }
```
Comment (SandraCoburn) [inline src/styles/less/ChildDashboard.less]: Are you gonna use that commented code on lines 108-11?

**Review by SandraCoburn — DISMISSED** [review]
Changes look great!

**Review by bramirez96 — DISMISSED** [review]
Looks great! B-

**Review by bramirez96 — DISMISSED** [review]
!

**Review by Schrese — DISMISSED** [review]
💯

**Review by Schrese — APPROVED** [review]

**Review by bramirez96 — APPROVED** [review]
Thanks for fixing those conflicts!

**Pull request merged**